src/main/java/in/org/projecteka/hiu/consent/ConsentException.java

```diff
@@ -0,0 +1,22 @@
+package in.org.projecteka.hiu.consent;
+
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+public class ConsentException extends Throwable {
+    private ErrorCode code;
+    private String message;
+
+    public static ConsentException creationFailed() {
+        final String failedToCreateRequest = "Failed to create consent request";
+        return new ConsentException(ErrorCode.CREATION_FAILED, failedToCreateRequest);
+    }
+
+    private enum ErrorCode {
+        CREATION_FAILED
```

Comment:
```suggestion
        CREATIONFAILED
```
https://google.github.io/styleguide/javaguide.html#s4.8.1-enum-classes Check whether it makes sense. If yes, follow the same for everything please.